src/main/java/mate/academy/spring/config/DataInitializer.java

```diff
@@ -26,10 +26,17 @@ public void inject() {
         Role userRole = new Role();
         userRole.setRoleName(Role.RoleName.USER);
         roleService.add(userRole);
-        User user = new User();
-        user.setEmail("admin@i.ua");
-        user.setPassword("admin123");
-        user.setRoles(Set.of(adminRole));
-        userService.add(user);
+
+        User admin = new User();
+        admin.setEmail("admin@i.ua");
+        admin.setPassword("admin123");
+        admin.setRoles(Set.of(adminRole));
```

Comment:
I meant adding the `USER` role to the admin user so that he would be able to call endpoints which are allowed for the usual users too, and not just the admin ones, so let's do that.
Alice can stay also as a test user, as long as it has some real email that belongs to us(if we were to develop some real application)